playbooks/roles/kibana/defaults/main.yml

```diff
@@ -1,8 +1,9 @@
 ---
-KIBANA_SERVER_NAME: "192.168.33.10"
+KIBANA_SERVER_NAME: "192.168.9.90" 
 KIBANA_NGINX_PORT: 80
 KIBANA_SSL_NGINX_PORT: 443
 
 kibana_app_dir: /edx/app/kibana
 kibana_file: kibana-3.0.0.tar.gz
-kibana_url: "https://download.elasticsearch.org/kibana/kibana/{{ kibana_file }}"
+kibana_version: kibana-3.0.0
```

Comment:
Now that we're parameterizing this, we should use the variable to build `kibana_file` like:

```yaml
kibana_version: '3.0.0'
kibana_file: "kibana-{{ kibana_version }}.tar.gz
```

Otherwise, if we upgraded we would need to change this version string in 2 places.